plugins/VstBase/RemoteVstPlugin32.cmake

```
@@ -34,7 +34,8 @@ ELSEIF(LMMS_BUILD_LINUX)
 		CMAKE_ARGS
 			"${EXTERNALPROJECT_CMAKE_ARGS}"
 			"-DCMAKE_CXX_COMPILER=${WINEGCC}"
-			"-DCMAKE_CXX_FLAGS=-m32"
+			"-DCMAKE_CXX_FLAGS=-m32 --winebuild \"${CUSTOM_WINEBUILD_EXECUTABLE}\""
```

```suggestion
			"-DCMAKE_CXX_FLAGS=-m32"
			"--winebuild \"${CUSTOM_WINEBUILD_EXECUTABLE}\""
```
Just keeping each argument on its own line

---

That breaks the build, actually; `--winebuild` isn't a CMAKE_ARG, it's part of the CMAKE_CXX_FLAGS :slightly_smiling_face: